models/base_model.py

```diff
@@ -70,6 +78,10 @@ def __init__(self, opt, rank):
             torch.backends.cudnn.benchmark = True
             torch.backends.cudnn.deterministic = False
         self.loss_names = []
+        self.loss_names_G = []
```

Comment:
I think adding all the loss names at one place in the code is better, I'd put this in `cut_model.py` & `cycle_gan_model.py` with the others.